content/overview/index.mdx

```diff
@@ -26,7 +26,7 @@ import {DocsTOC} from '~/components/DocsTOC'
 
 Welcome to Depot!
 
-Depot is a remote container build service that makes image builds up to 20x faster than building Docker images inside generic CI providers. Docker image builds get sent to a fast builder instance with a persistent cache. The resulting image can then be downloaded locally or pushed to a registry. Adopting Depot is easy, as the Depot CLI `depot build` command accepts the same arguments as `docker buildx build`.
+Depot is a remote container build service that makes image builds up to [80x](https://github.com/depot/benchmark-mastodon/actions/runs/4893869663) faster than building Docker images inside generic CI providers. Docker image builds get sent to a fast builder instance with a persistent cache. The resulting image can then be downloaded locally or pushed to a registry. Adopting Depot is easy, as the Depot CLI `depot build` command accepts the same arguments as `docker buildx build`.
```

Comment:
For this, I think we want to be careful about what speedup we mention, since most customer projects are closer to the 20x, Mastodon is definitely the greatest speedup we've seen in the wild. But it's definitely a good idea to link to the fast benchmark. 🙂 

```suggestion
Depot is a remote container build service that makes image builds up to 20x faster than building Docker images inside generic CI providers (with some projects showing [even faster speedups](https://depot.dev/benchmark/mastodon)). Docker image builds get sent to a fast builder instance with a persistent cache. The resulting image can then be downloaded locally or pushed to a registry. Adopting Depot is easy, as the Depot CLI `depot build` command accepts the same arguments as `docker buildx build`.
```